src/index.html

```diff
@@ -7,8 +7,34 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,400;1,500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="nav">
+        <ul class="list_of_nav_links">
+          <li><a href="#" class="nav_link is-active">apple</a></li>
+          <li><a href="#" class="nav_link">samsung</a></li>
+          <li><a href="#" class="nav_link">smartphones</a></li>
+          <li>
+            <a href="#" 
+               class="nav_link" 
+               data-qa="hover">laptops & computers
```

Comment:
```suggestion
            <a 
              href="#" 
              class="nav_link" 
              data-qa="hover"
            >
              laptops & computers
```

Reply:
Move every attribute to new line with +2 indentation if you have more than 2 attrs